Feature/mtm 60035/sso alternative flow (#2327)
SoftwareAG/c8y-docs: content/authentication/sso-bundle/integration-with-azure.md

1. Select **App Registrations** under **Manage** on the left and at the top click **New Registration**.
3. In the resulting window, provide a name for the new App registration.
4. As **Redirect URI type** select "Web" and enter the URL to your tenant OAuth endpoint, for example "https:&#47;/documentation.cumulocity.com/tenant/oauth". You can derive this value from your {{< product-c8y-iot >}} tenant. Navigate to **Administration** > **Settings** > **Authentication** > **Single sign-on**. The redirect URL is prefilled by the platform. 

> Optionally set e.g. "https:&#47;/documentation.cumulocity.com/apps/*" if in {{< product-c8y-iot >}} SSO configuration `Redirect to the user interface application` is enabled.

Reviewer: ```suggestion
Optionally, set for example, "https:&#47;/documentation.cumulocity.com/apps/*" if in the {{< product-c8y-iot >}} SSO configuration `Redirect to the user interface application` is enabled.
```

> Optionally set e.g. "https:&#47;/documentation.cumulocity.com/apps/*" if in {{< product-c8y-iot >}} SSO configuration `Redirect to the user interface application` is enabled.

Reviewer 2: 1. Instead of hard coding `/documentation.cumulocity.com/apps/*` can we keep it generic like `"<tenant_domain>/apps/*"` 
2. Can you change SSO configuration to SSO server ? because we set this in SSO server and it will be inline with the release notes

Author: Updated tenant_domain, rather here we talking about SSO configuration, not sso server.
This 'Redirect to the user interface application' is set in Cumulocity SSO configuration.
